metricflow-semantics/tests_metricflow_semantics/snapshots/test_query_parser.py/str/test_join_through_scd_no_time_dimension_validation__error.txt

```diff
@@ -0,0 +1,17 @@
+Got errors while resolving the query.
+
+Error #1:
+  Message:
+
+    Your query contains the group bys which are SCDs or contain SCDs in the join path: [listing__user__home_state_latest].
```

Comment:
Nit on the message here - can we say (changes in code syntax):
Your query contains the `following` group bys`,` which are SCDs or contain SCDs in the join path: [`'`listing__user__home_state_latest`'`].